app/move/controllers/view.js

```diff
@@ -14,6 +15,21 @@ module.exports = function view(req, res) {
     cancellationReason === 'other'
       ? cancellationComments
       : req.t(`fields::cancellation_reason.items.${cancellationReason}.label`)
+
+  const urls = {
```

Comment:
Maybe name it something more clear that these are edit URLs?

Reply:
But it's an object of all urls. That only currently has update urls in it, but seems cleaner to say `urls.update.foo` than `updateUrls.foo`